spec/integration/hawkular_client_spec.rb

```diff
@@ -267,6 +267,18 @@ module Hawkular::Client::RSpec
         end
       end
 
+      it 'Should work initializing with URI' do
+        uri = URI.parse HOST
+        opts = { tenant: 'hawkular' }
+        WebSocketVCR.configure do |c|
+          c.hook_uris = [HOST]
+        end
+        WebSocketVCR.record(example, self) do
+          the_client = Hawkular::Client.new(entrypoint: uri, credentials: @creds, options: opts)
+          expect { the_client.operations }.to_not raise_error
```

Comment:
Here you need a running server to test/do the initial recording. The cassette provided is empty. 
Perhaps @Jiri-Kremser can help you if needed.